Bottom: 8c6f51e3b7ebfcde94689c820e984c0be013ced2
Top:    53565c7d53406cff640536eda2a06158558e15d1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:54:21 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index 49f011f..1c1a504 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
